Floofbot/Modules/Administration.cs

```diff
@@ -65,25 +67,33 @@ public class Administration : InteractiveBase
                 }
             }
 
-            //sends message to user
-            EmbedBuilder builder = new EmbedBuilder();
-            builder.Title = "⚖️ Ban Notification";
-            builder.Description = $"You have been banned from {Context.Guild.Name}";
-            builder.AddField("Reason", reason);
-            builder.Color = ADMIN_COLOR;
-            await badUser.SendMessageAsync("", false, builder.Build());
+            try
+            {
+                //sends message to user
```

Comment:
Best practice is typically to limit the try/catch block to only the section of code that raises the exception. But I'll let this slide